### PR TITLE
Add title to toolbar plugins

### PR DESCRIPTION
**Pull request by always-maap**
add a title with a shortcut to toolbar plugins

Screen Shot 1401-02-02 at 14 13 11

### CODE REVIEW

File: packages/lexical-playground/src/utils/toolbar-shortcut.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const isApple = /(Mac|iPhone|iPad)/i.test(navigator.platform);
```
Comment (thegreatercurve) [inline packages/lexical-playground/src/utils/toolbar-shortcut.js:1]: We have a utility called `IS_APPLE` in `packages/shared`  which does the same thing.

File: packages/lexical-playground/src/utils/toolbar-shortcut.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+export const toolbarShortcut = {
+  bold: isApple ? 'Bold (⌘B)' : 'Bold (Ctrl+B)',
+  italic: isApple ? 'Italic (⌘I)' : 'Italic (Ctrl+I)',
+  redo: isApple ? 'Redo (⌘Y)' : 'Undo (Ctrl+Y)',
+  underline: isApple ? 'Underline (⌘U)' : 'Underline (Ctrl+U)',
+  undo: isApple ? 'Undo (⌘Z)' : 'Undo (Ctrl+Z)',
+};
```
Comment (thegreatercurve) [inline packages/lexical-playground/src/utils/toolbar-shortcut.js:9]: Can we put these inline as props in `packages/lexical-playground/src/plugins/ToolbarPlugin.jsx`? 

There's probably not much point in abstracting them into a utils file if we're not reusing them elsewhere.

**Review by thegreatercurve — COMMENTED** [review]
Thanks for spotting and making these changes @always-maap! LGTM but just left a couple of comments.

**Comment by always-maap** [comment]
> Thanks for spotting and making these changes @always-maap! LGTM but just left a couple of comments.

fixed your comments

**Review by zurfyx — APPROVED** [review]
Thanks!

Just noticed @thegreatercurve's comments, want to have another pass yourself and merge?

**Review by thegreatercurve — APPROVED** [review]
Thanks @always-maap!

**Comment by acywatson** [comment]
Overall, this looks like an improvement, but I was hoping we could make it accessible. See the thread here:

https://github.com/facebook/lexical/issues/1726

@zurfyx @thegreatercurve

**Pull request merged**